packages/eslint-plugin/tests/rules/no-duplicate-type-constituents.test.ts

```diff
@@ -275,19 +278,6 @@ type ActuallyDuplicated = IsArray<number>  ;
         },
       ],
     },
-    {
-      code: 'type T = Class<string> | Class<string>;',
-      output: `type T = Class<string>  ;`,
-      errors: [
-        {
-          messageId: 'duplicate',
-          data: {
-            type: 'Union',
-            previous: 'Class<string>',
-          },
-        },
-      ],
-    },
```

Comment:
[Testing] If this case is no longer reported as an error, let's just move it to the `valid` section instead of deleting it?

Reply:
Indeed, that's correct. I moved the tests in [3cf71ad](https://github.com/typescript-eslint/typescript-eslint/pull/9600/commits/3cf71add098ede30ecd2ace0b4c857737ca65ded).